exporter/abstractions_test.go

```diff
@@ -0,0 +1 @@
+package exporter
```

Comment:
Is this placeholder file intentional? If there are no tests yet for abstractions.go, might be worth either adding tests for `copyEffectiveFieldsToInputFieldsWithConverters` or removing the empty file.